scripts/tests/smoke-test-gui-linux.sh

```diff
@@ -65,11 +65,20 @@ function crash_test() {
     sudo rm "$DUMP_PATH"
 }
 
+function get_stacktrace() {
+    SYMS_PATH="../target/debug/firezone-gui-client.syms"
+    cargo install --locked dump_syms minidump-stackwalk
+    dump_syms ../target/debug/firezone-gui-client.dwp --output "$SYMS_PATH"
+    ls -lash ../target/debug
+    minidump-stackwalk --symbols-path "$SYMS_PATH" "$DUMP_PATH"
```

Comment:
I'm not getting a very good stacktrace from this on Linux, but I am getting something.